tests/waku_rln_relay/rln/test_wrappers.nim

```diff
@@ -0,0 +1,131 @@
+import testutils/unittests
+
+import
+  ../../../waku/[waku_rln_relay, waku_rln_relay/rln, waku_rln_relay/rln/wrappers],
+  ./waku_rln_relay_utils,
+  ../../testlib/[simple_mock]
+
+import std/[options]
+import
+  chronicles,
+  chronos,
+  stew/shims/net as stewNet,
+  eth/keys,
+  bearssl,
+  stew/[results],
+  metrics,
+  metrics/chronos_httpserver
+
+const Empty32Array = default(array[32, byte])
+proc valid(x: seq[byte]): bool =
```

Comment:
super nitpick, but better to have an empty line before each proc

Reply:
Done